docs/what-we-look-at.md

```diff
@@ -0,0 +1,48 @@
+# What we look at
+
+Licensee works by taking a detected license file, and comparing the contents to a short list of known licenses.
+
+## Detecting the license file
+
+Licensee uses [a series of regular expressions](https://github.com/benbalter/licensee/blob/master/lib/licensee/project_files/license_file.rb#L15-L25) to score files in the project's root as potential license files. Here's a few examples of files that would be detected:
+
+* `LICENSE`
+* `LICENCE`
+* `license.md`
+* `COPYING.txt`
+* `LICENSE-MIT`
+* `COPYRIGHT`
+* `UNLICENSE`
+
+## Known licenses
+
+Licensee relies on the crowdsourced license content and metadata from [`choosealicense.com`](http://choosealicense.com).
+
+## What it doesn't look at
+
+* The licensing of a project's dependencies
+* References to licenses in `README`, `README.md`, etc.
+* Every single possible license (just the most popular ones)
+* Compliance - If you're looking for this, take a look at [LicenseFinder](https://github.com/pivotal/LicenseFinder).
+
+## Huh? Why don't you look at X?
+
+Because reasons.
+
+### Why not just look at the "license" field of [insert package manager here]?
+
+Because it's not legally binding. A license is a legal contract. You give up certain rights (e.g., the right to sue the author) in exchange for the right to use the software.
+
+Most popular licenses today *require* that the license itself be distributed along side the software. Simply putting the letters "MIT" or "GPL" in a configuration file doesn't really meet that requirement.
+
+Not to mention, it doesn't tell you much about your rights as a user. Is it GPLv2? GPLv2 or later? Those files are designed to be read by computers (who can't enter into contracts), not humans (who can). It's great metadata, but that's about it.
+
+### What about looking to see if the author said something in the readme?
+
+You could make an argument that, when linked or sufficiently identified, the terms of the license are incorporated by reference, or at least that the author's intent is there. There's a handful of reasons why this isn't ideal. For one, if you're using the MIT or BSD (ISC) license, along with a few others, there's templematic language, like the copyright notice, which would go unfilled.
+
+### What about checking every single file for a copyright header?
+
+Because that's silly in the context of how software is developed today. You wouldn't put a copyright notice on each page of a book. Besides, it's a lot of work, as there's no standardized, cross-platform way to describe a project's license within a comment.
```

Comment:
This is a great argument (I still remember reading [the blog post](http://ben.balter.com/2015/06/03/copyright-notices-for-websites-and-open-source-projects/) where you first (AFAIK) described it), but I'd refrain on the "silly". Even "Because that doesn't make sense ..." would be friendlier.

I'd also replace the "it's a lot of work" (which may sound like laziness) with something like "it's very hard to do right" (which matches the rest of the sentence).